content/dev/pmc.md

```diff
@@ -409,7 +409,9 @@ although PMCs are free to follow their own documented process for finding
 consensus on adding new committers.
 
 Once the PMC formally wants to invite an individual to be a committer, 
-it should invite the person, and require the new committer to 
+it should invite the person, 
+([use this template](https://community.apache.org/newcommitter.html#committer-invite-template))
+and require the new committer to 
 [submit an Individual Contributor License Agreement (ICLA) to the secretary](/licenses/#clas). 
```

Comment:
... if an ICLA is not already on file.

Reply:
FYI @clr-apache -> when you click (+) on the modified line (or even select few lines) - you can make a suggestion to change in the PR - this is cool, because the author can just click "commit suggestion" and also you are automatically marked as co-author of that change.

I just added your comment as suggestion :)